### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by SeanFeldman**
Minor fix

### CODE REVIEW

File: src/Microsoft.Azure.ServiceBus/Primitives/DispositionStatus.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Azure.ServiceBus.Primitives
     enum DispositionStatus
```
Comment (nemakam) [inline src/Microsoft.Azure.ServiceBus/Primitives/DispositionStatus.cs]: Check out the comments on top of this enum.

**Comment by SimonCropp** [comment]
@nemakam so where is the plan fix this long term?

**Comment by nemakam** [comment]
@SimonCropp , we have that as a backlog item in our service side tasks. Once that is changed, we will update all the clients. However, for backward compatibility, it would be difficult to stop supporting the typo'd version.

**Comment by SimonCropp** [comment]
> However, for backward compatibility, it would be difficult to stop supporting the typo'd version.

true. but you could also support the correct spelling moving forward

**Comment by SeanFeldman** [comment]
Adding more descriptive comment might add clarity.​

**Comment by SimonCropp** [comment]
>  we have that as a backlog item in our service side tasks. Once that is changed, we will update all the clients

can u share the url for that issue

**Comment by nemakam** [comment]
> can u share the url for that issue

We track and plan our work in VSTS (VisualStudio.com) and not github. Unfortunately, it doesn't support anonymous access.

**Comment by SimonCropp** [comment]
@nemakam then you should be clear about that and remove the issues tab from this github repo.

**Pull request closed without merging**